### ISSUE TITLE
Alternatives to Tensorflow

### ISSUE THREAD

**Issue by alexklibisz** [alexklibisz/tensortube#6]
@gbruer15 mentioned this at some point. It might be interesting to hook in with some alternatives to the Tensorflow / Imagenet neural net for the purpose of comparing performance (compute and accuracy). 

There are also some external APIs / services we could hit, specifically [Google Vision API](https://cloud.google.com/vision/) and [Clarifai](https://www.clarifai.com/)
